packages/rollup-full-node/test/app/web-rpc-handler.spec.ts

```diff
@@ -93,6 +97,41 @@ describe('Web3Handler', () => {
     }
   })
 
+  describe('the getBlockByNumber endpoint', () => {
+    it('should return a block with the correct timestamp', async () => {
+      const httpProvider = new ethers.providers.JsonRpcProvider(baseUrl)
+      const block = await httpProvider.getBlock('latest')
+
+      block.timestamp.should.be.gt(0)
+    })
+
+    it('should strip the execution manager deployment transaction from the transactions object', async () => {
+      const httpProvider = new ethers.providers.JsonRpcProvider(baseUrl)
+      const block = await httpProvider.getBlock(1, true)
+
+      block['transactions'].should.be.empty
+    })
+
+    it('should increase the timestamp when blocks are created', async () => {
+      const httpProvider = new ethers.providers.JsonRpcProvider(baseUrl)
+      const executionManagerAddress = await httpProvider.send(
+        'ovm_getExecutionManagerAddress',
+        []
+      )
+      const { timestamp } = await httpProvider.getBlock('latest')
+      const wallet = getWallet(httpProvider)
+      const simpleStorage = await deploySimpleStorage(wallet)
+      await setAndGetStorage(
+        simpleStorage,
+        httpProvider,
+        executionManagerAddress
+      )
+
+      const block = await httpProvider.getBlock('latest', true)
+      block.timestamp.should.be.gt(timestamp)
```

Comment:
one note is that if we do end up speeding up transactions enough, then we could end up with `block.timestamp` equaling `timestamp`

Reply:
Ah good call! I [changed it to greater than or equal to](https://github.com/ethereum-optimism/optimism-monorepo/pull/59/commits/b081e7db74d1615ad82e2c0d2a022f558545c363) just incase.